LayoutDemo/app/src/main/java/edu/h2/layoutdemo/data/service/ChangePasswordServiceImpl.java

```diff
@@ -0,0 +1,11 @@
+package edu.h2.layoutdemo.data.service;
+
+import edu.h2.layoutdemo.domain.service.ChangePasswordService;
+import io.reactivex.Observable;
+
+public class ChangePasswordServiceImpl implements ChangePasswordService {
+    @Override
+    public Observable<Boolean> requestChangePassword(String driverId, String oldPassword, String newPassword, String confirmNewPassword) {
```

Comment:
Method name: changePassword is fine.
We do not need confirmNewPassword parameter